### PR TITLE
Deploy confirmation

### PR DESCRIPTION
**Pull request by alexandreesl**
## What changes were proposed in this pull request?

This PR is intended to add a change on the way Nerdployer work with K8S, making him wait for deployments to complete, instead of fire-and-forget.

## How was this patch tested?

This patch was tested with recommendations stack on QA environment.

Ref:. https://github.com/grupozap/squad-recommendation/issues/241

### CODE REVIEW

File: nerdployer/helpers/k8s.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import subprocess
```
Comment (cdmbr) [inline nerdployer/helpers/k8s.py]: all these heavy lifting config, parameter binding should be on the step side not on the helpers side ... helpers  in this project are not aware of context, config and etc .... plz check out others helpers like the cloudformation one to see the pattern that we have followed here ...

Comment (alexandreesl) [reply nerdployer/helpers/k8s.py]: please look again :-)

File: nerdployer/helpers/k8s.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        self.opts = opts
+
+    def apply(self, template):
+        apply_command = 'cat <<EOF | kubectl --server={server} --token={token} {namespace} {opts} ' \
+                        'apply -f -\n{template}\nEOF'.format(
+            server=self.server, token=self.token, template=template, opts=self.opts, namespace=self.namespace)
+        return self._run_sub_process(apply_command)
+
+    def rollout_status(self, deployment):
+        check_command = 'kubectl --server={server} --token={token} {namespace} ' \
```
Comment (cdmbr) [inline nerdployer/helpers/k8s.py]: missing opts

File: nerdployer/helpers/k8s.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import subprocess
+
+
+class K8s(object):
+
+    def __init__(self, server, token, namespace, opts):
+        self.server = server
```
Comment (cdmbr) [inline nerdployer/helpers/k8s.py]: what about building a base cmd str here ... 
self.base_cmd = 'kubectl --server={server} --token={token} {namespace} {ops}'

**Pull request merged**